src/Og.php

```diff
@@ -169,6 +169,32 @@ public static function getEntityGroups($entity_type, $entity_id, $states = [OG_S
   }
 
   /**
+   * Return TRUE if entity belongs to a group.
+   *
+   * @param string $group_entity_type
+   *   The entity type of the group.
+   * @param string|int $group_id
+   *   The group ID.
+   * @param string $entity_type
+   *   The entity type.
+   * @param string|int $entity_id
+   *   The entity ID.
+   * @param array $states
+   *   (optional) Array with the state to return. If empty groups of all state
+   *   will return.
+   *
+   * @return bool
+   *   TRUE if the entity (e.g. the user) belongs to a group and is not pending
+   *   or blocked.
+   */
+  public static function isMember($group_entity_type, $group_id, $entity_type, $entity_id = NULL, $states = [OG_STATE_ACTIVE]) {
```

Comment:
I think maybe it's a good time to replace the arguments with the entity objects, so it will be `isMember(EntityInterface $group_entity, EntityInterface $entity)` instead. Thoughts?